.github/workflows/release-please.yml

```diff
@@ -0,0 +1,24 @@
+name: Release Please
+
+on:
+  schedule:
+    - cron: "0 13 * * 1"
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release-please:
+    if: ${{ github.repository == 'opencadc/deployments' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Release Please
+        uses: google-github-actions/release-please-action@v4
+        with:
+          command: manifest
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Out of scope for this PR, the release process is being handled in separate work and has slightly different requirements than this work.